modules/ROOT/pages/db/db-connector-execute-script-ref.adoc

```diff
@@ -47,4 +44,9 @@ data into historic tables and purge on-line tables.
 
 Execute script and bulk operations are intended for different uses.  For example, although you can build a script that inserts many rows into the database, you could not conveniently provide dynamic parameters to the script. Also, you need to ensure that the script protects against a SQL injection attack.
 
-Executing a select statement in a script, although possible, returns no data and poses a data input problem; therefore, doing so makes no sense.
\ No newline at end of file
+Executing a select statement in a script, although possible, returns no data and poses a data input problem; therefore, doing so makes no sense.
```

Comment:
I would delete the last part--just say don't do it because it causes a data problem.

Reply:
Fixed as requested